CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Allow in-memory synced Realms. This will allow setting and in-memory identifier on
+  flexible sync realms, this will not create a .realm file or its associated auxiliary 
+  files for the synced realm and instead stores objects in memory while the realm is 
+  open and discards them immediately when all instances are closed.
```

Comment:
This isn't an accurate description of in-memory Realms. We still create a file and just delete it when it's closed, and never sync to disk so it's unlikely to be actually written.